modules/@apostrophecms/page/ui/apos/components/AposPagesManager.vue

```diff
@@ -241,6 +258,27 @@ export default {
         this.checkedDocs.push(doc);
         this.checked.push(doc._id);
       }
+    },
+    updateCheckedDocs() {
+      this.checkedDocs = this.checked.map(_id => find([ this.originalPageTree ], item => {
```

Comment:
It looks like we could simply put the unadulterated page object into `pagesFlat` and then use that instead. No need for the additional `find` method.